data/data/bootstrap/baremetal/files/etc/ironic.env.template

```diff
@@ -0,0 +1,9 @@
+AUTH_DIR=/opt/metal3/auth
+HTTP_PORT=6180
+# This DHCP range is used by dnsmasq to serve DHCP to the cluster. If empty
+# dnsmasq will only serve TFTP, and DHCP will be disabled.
+DHCP_RANGE="{{.PlatformData.BareMetal.ProvisioningDHCPRange}}"
+DHCP_ALLOW_MACS="{{.PlatformData.BareMetal.ProvisioningDHCPAllowList}}"
+# Used by ironic to allow ssh to running IPA instances
+IRONIC_RAMDISK_SSH_KEY="{{.SSHKey}}"
+IPTABLES={{if .PlatformData.BareMetal.ProvisioningIPv6}}ip6tables{{else}}iptables{{end}}
```

Comment:
I'd rather prefer everything to be handled by built-ironic-env.sh rather than having two places to keep in mind.

Reply:
Why would you have problems with shellcheck? From its perspective, `IRONIC_RAMDISK_SSH_KEY="{{.SSHKey}}"` is just setting a variable to a string. I don't think bash interprets {{ in strings?

Reply:
You have to rename the file to `.sh.template`